### PR TITLE
chore: refactor helpers

### PR DESCRIPTION
**Pull request by heueristik**
After the changes on https://github.com/aragon/osx-commons/pull/72 and https://github.com/aragon/osx-commons/pull/74, we want to bring them into the template.

Task-ID: [OS-1273](https://aragonassociation.atlassian.net/browse/OS-1273)

[OS-1273]: https://aragonassociation.atlassian.net/browse/OS-1273?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/contracts/test/20_integration-testing/21_deployment.ts [hunk 1]
```diff
@@ -80,7 +79,9 @@ describe(`Deployment on network '${productionNetworkName}'`, function () {
         JSON.stringify(METADATA.build, null, 2)
       )}`;
 
-      expect(results.buildMetadata).to.equal(toHex(buildMetadataURI));
+      expect(results.buildMetadata).to.equal(
+        ethers.utils.hexlify(ethers.utils.toUtf8Bytes(buildMetadataURI))
```
Comment (josemarinas) [inline packages/contracts/test/20_integration-testing/21_deployment.ts:83]: `toHex`?

Comment (heueristik) [reply packages/contracts/test/20_integration-testing/21_deployment.ts:83]: What do you mean? We removed `toHex`

**Review by josemarinas — APPROVED** [review]

File: packages/subgraph/schema.graphql
Comment (clauBv23) [inline packages/subgraph/schema.graphql:1]: what is the convention for having or not this new line at the end of a file?

**Review by clauBv23 — APPROVED** [review]

File: packages/subgraph/schema.graphql
Comment (heueristik) [reply packages/subgraph/schema.graphql:1]: There were two empty lines and we want to have one

Comment (clauBv23) [reply packages/subgraph/schema.graphql:1]: sorry didn’t notice it

**Pull request merged**